types/config.go

```diff
@@ -81,6 +85,15 @@ func (config *Config) SetAddressVerifier(addressVerifier func([]byte) error) {
 	config.addressVerifier = addressVerifier
 }
 
+func (config *Config) SetCoinType(coinType uint32) {
+	config.assertNotSealed()
+	config.coinType = coinType
+}
+
+func (config *Config) SetFullFundraiserPath(fullFundraiserPath string) {
+	config.fullFundraiserPath = fullFundraiserPath
```

Comment:
Why do we not call `assertNotSealed` here?

Reply:
It's my mistake.
Fixed